crates/nu-command/src/conversions/into/binary.rs

```diff
@@ -39,6 +40,7 @@ impl Command for SubCommand {
                 (Type::Record(vec![]), Type::Record(vec![])),
             ])
             .allow_variants_without_examples(true) // TODO: supply exhaustive examples
+            .switch("compact", "output without padding zeros", None)
```

Comment:
Instead of `None` here, it seems like we should have a shortcut like `Some('c')`